DOKS-wordpress/README.md

```diff
@@ -266,6 +522,87 @@ NAME                                   DESIRED   CURRENT   READY   AGE
 replicaset.apps/wordpress-6f55c9ffbd   1         1         1       23h
 ```
 
+Verify the PVSs created under the wordpress namespace and associated OpenEBS volume under the openebs namespace:
+
+```console
+kubectl get pvc -A
+```
+
+The ouput looks similar to (notice the `RWX` access mode for the wordpress PVC and it's storage class created earlier when configuring the openEBS NFS provisioner):
+
+```text
+NAMESPACE   NAME                                           STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+openebs     nfs-pvc-b505c0af-e6ab-4623-8ad1-1bad784261d5   Bound    pvc-d3f0c597-69ba-4710-bd7d-ed29ce41ce04   5Gi        RWO            do-block-storage   20m
+wordpress   wordpress                                      Bound    pvc-b505c0af-e6ab-4623-8ad1-1bad784261d5   5Gi        RWX            rwx-storage        20m
+```
+
+Verify the associated PVs created in the cluster:
+
+```console
+kubectl get pv
+```
+
+The ouput looks similar to:
+
+```text
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                                  STORAGECLASS       REASON   AGE
+pvc-b505c0af-e6ab-4623-8ad1-1bad784261d5   5Gi        RWX            Delete           Bound    wordpress/wordpress                                    rwx-storage                 23m
+pvc-d3f0c597-69ba-4710-bd7d-ed29ce41ce04   5Gi        RWO            Delete           Bound    openebs/nfs-pvc-b505c0af-e6ab-4623-8ad1-1bad784261d5   do-block-storage            23m
+```
+
+We can also create additional pod replicas to demonstrate the power of the NFS provisioner by opening the `values.yaml` file and uncommenting the: `replicaCount: 3` line (this can be set to as many replicas as you wish).
```

Comment:
Avoid using `we`, `our`, etc. And, maybe reformulate `demonstrate the power` to `demonstrate the capabilities`.